examples/asio-grpc/boo.cpp

```diff
@@ -0,0 +1,4 @@
+#include <agrpc/asioGrpc.hpp>
+
+int main() {
+}
```

Comment:
could you use something from the header here, to make it double sure, that the include did work?